### ISSUE TITLE
Tests are unmapped when test file has angle bracket Typescript cast syntax

### ISSUE THREAD

**Issue by dmcweeney** [lucono/karma-test-explorer#49]
**Describe the Bug**

Tests that use the older Typescript cast syntax **`<type>`** result in TestLocator parsing error and the tests in the file get rendered as Unmapped in the test Explorer

`[ERROR] [TestLocator]: Failed to get tests from spec file sample.spec.ts: SyntaxError: Unexpected token, expected "}"`

**Steps to Reproduce the Behavior**

```
interface ISample {
  id: string;
  other: string;
}

describe('Outer Tests', () => {
  const sampleId: unknown = 12345;
  const sampleArray: unknown[] = [sampleId];

  it('this should trigger SyntaxError: Unexpected token', () => {
    // arrange
    const data: ISample = {
      id: <string>sampleId,
      other: 'value'
    };
    // act
    expect(data).toBeNull();
  });

  sampleArray.forEach(testValue => {
    it('with ' + testValue + ' should trigger SyntaxError: Unexpected token', () => {
      // arrange
      const data: ISample = {
        id: <string>testValue,
        other: 'value'
      };
      // act
      expect(data).toBeNull();
    });
  });
});

```

**Provide Debug Logs**

This is as much detail as there is
`[ERROR] [TestLocator]: Failed to get tests from spec file sample.spec.ts: SyntaxError: Unexpected token, expected "}"`

**Please Provide the Following Information**
 
 - OS: Windows 11
 - Environment: dev box
 - Node Version: v16.15.0
 - VS Code Version:  1.70.2
 - Karma Version: v6.3.5
 - Frameworks: Jasmine

**Additional Context**

Tests/parsing work fine when running tests via command: `karma start karma.conf.js`

From package.json

```
  "devDependencies": {
    "@commitlint/cli": "^14.1.0",
    "@commitlint/config-angular": "^14.1.0",
    "@types/copy-webpack-plugin": "^8.0.1",
    "@types/jasmine": "^3.10.0",
    "@types/lodash.clonedeep": "^4.5.6",
    "@types/node": "^16.9.6",
    "@types/webpack": "^5.28.0",
    "@typescript-eslint/eslint-plugin": "^5.3.0",
    "@typescript-eslint/parser": "^5.3.0",
    "clean-webpack-plugin": "^4.0.0",
    "commitizen": "^4.2.4",
    "copy-webpack-plugin": "^9.0.1",
    "eslint": "^8.1.0",
    "eslint-config-prettier": "^8.3.0",
    "eslint-plugin-jasmine": "^4.1.3",
    "eslint-plugin-prettier": "^4.0.0",
    "husky": "^7.0.4",
    "istanbul-instrumenter-loader": "^3.0.1",
    "jasmine-core": "^4.2.0",
    "jasmine-spec-reporter": "^7.0.0",
    "karma": "^6.3.5",
    "karma-chrome-launcher": "^3.1.0",
    "karma-coverage": "^2.0.3",
    "karma-coverage-istanbul-reporter": "^3.0.3",
    "karma-jasmine": "^4.0.1",
    "karma-jasmine-html-reporter": "^1.7.0",
    "karma-webpack": "^5.0.0",
    "make-dir-cli": "^3.0.0",
    "prettier": "^2.4.1",
    "tsconfig-paths": "^4.1.0",
    "ts-loader": "^9.2.6",
    "ts-node": "^10.9.1",
    "tslib": "^2.3.1",
    "typescript": "^4.4.3",
    "webpack": "^5.59.1",
    "webpack-cli": "^4.9.1"
  },

```

**Comment by lucono** [comment]
@dmcweeney Try out the fix in the beta branch to confirm if it resolves the issue. You can use the [vsix](https://github.com/lucono/karma-test-explorer/suites/7988521944/artifacts/342039976) from the CI build, or build it yourself from the `beta` branch.

**Comment by dmcweeney** [comment]
@lucono that worked - thank you.

**Comment by lucono** [comment]
Released in [v0.7.4](https://github.com/lucono/karma-test-explorer/blob/master/CHANGELOG.md#074---aug-27-2022)

**Comment by Cogur** [comment]
I've just downloaded the latest version but still experincing this issue (I gues)

[2022-08-30 08:07:06.608] [ERROR] [TestLocator]: Failed to get tests from spec file C:/Users/***.component.spec.ts: SyntaxError: Unexpected token (10:23)
 but the test file works perfectly anyway.
 
 my current version is 0.7.4
